codespell_lib/data/dictionary.txt

```diff
@@ -56938,6 +56938,7 @@ timestemp->timestamp
 timestemps->timestamps
 timestmap->timestamp
 timestmaps->timestamps
+timestmp->timestamp
```

Comment:
How about adding `timestmps->timestamps` too?

Reply:
Perhaps also variation timestamped, timestamping.

Reply:
Appreciate it, @skangas !
I have added the variations you suggested.